pkg/data/values_test.go

```diff
@@ -0,0 +1,108 @@
+package data
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetValue(t *testing.T) {
+	tests := []struct {
+		name        string
+		data        map[string]interface{}
+		keys        []string
+		wantValue   interface{}
+		wantSuccess bool
+	}{
+		{
+			name:        "nil map",
+			data:        nil,
+			keys:        []string{"somekey"},
+			wantValue:   nil,
+			wantSuccess: false,
+		},
+		{
+			name: "key is not in map",
+			data: map[string]interface{}{
+				"realKey": "realVal",
+			},
+			keys:        []string{"badKey"},
+			wantValue:   nil,
+			wantSuccess: false,
+		},
+		{
+			name: "key is in first level of map",
+			data: map[string]interface{}{
+				"realKey": "realVal",
+			},
+			keys:        []string{"realKey"},
+			wantValue:   "realVal",
+			wantSuccess: true,
+		},
+		{
+			name: "key is nested in map",
```

Comment:
No, there is no valid path for parent -> grandhild -> child, it will stop processing when it doesn't find "grandchild" under "parent". Adding a test to show this.